spec/system/service_unavailable_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Service Unavailable' do
+  let(:user) { create :user }
+
+  context 'when the service is unavailable' do
+    before do
+      allow(ENV).to receive(:[]).and_call_original
+      allow(ENV).to receive(:[]).with('SERVICE_UNAVAILABLE').and_return('true')
+      visit '/'
```

Comment:
We could also assert that a couple of pages all get redirected, or that a loop isn't possible, with little effort.

Reply:
[796f7c7](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/941/commits/796f7c7bfd6504680528e11e99c3cbeaee5a27d0)